scripts/jenkins/get_remote_build.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+"""
+This script retreives the latest successful build from
+apm-ci.elastic.co and returns the commit hash
+"""
+import os
+from jenkinsapi.jenkins import Jenkins
+
+def getSCMInfroFromLatestGoodBuild(url, jobName, username=None, password=None):
```

Comment:
I guess it's a typo: "getSCM`Infro`FromLatestGoodBuild"

Reply:
You're right! Thanks for catching this.